### PR TITLE
Update FAQ-journey.yml

### PR DESCRIPTION
**Pull request by O365scripts**
Typo in upgraded and also with inverted bracket in skype to teams journey link.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/FAQ-journey.yml [hunk 1]
```diff
@@ -138,4 +138,4 @@ sections:
       
       - question: Will the latest generation of Skype Room Systems v2 support meetings in Teams?
         answer: | 
-         We rebranded Skype Room Systems to Microsoft Teams Rooms which fully supports Microsoft Teams meetings and offer an easy migration path from Skype for Business to Teams by just enabling Teams on the device. In addition to the ability for users to identify nearby Microsoft Teams Rooms with proximity detection, Teams meetings can be joined with a single-click, dual screen support, Microsoft Whiteboard and we continue to bring innovative features like content camera with intelligent capture.
\ No newline at end of file
+         We rebranded Skype Room Systems to Microsoft Teams Rooms which fully supports Microsoft Teams meetings and offer an easy migration path from Skype for Business to Teams by just enabling Teams on the device. In addition to the ability for users to identify nearby Microsoft Teams Rooms with proximity detection, Teams meetings can be joined with a single-click, dual screen support, Microsoft Whiteboard and we continue to bring innovative features like content camera with intelligent capture.
```
Comment (JohanFreelancer9) [inline Teams/FAQ-journey.yml:141]:
```suggestion
         We rebranded Skype Room Systems to Microsoft Teams Rooms which fully supports Microsoft Teams meetings and offers an easy migration path from Skype for Business to Teams by just enabling Teams on the device. In addition to the ability for users to identify nearby Microsoft Teams Rooms with proximity detection, Teams meetings can be joined with a single-click, dual-screen support, Microsoft Whiteboard and we continue to bring innovative features like a content camera with intelligent capture.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few additional changes, please.

**Review by russparker37 — APPROVED** [review]

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**